ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManagerImpl.java

```diff
@@ -342,8 +342,12 @@ public void onPackagerStatusFetched(final boolean packagerIsRunning) {
                       public void run() {
                         if (packagerIsRunning) {
                           mDevSupportManager.handleReloadJS();
-                        } else {
+                        } else if (mDevSupportManager.hasUpToDateJSBundleInCache()) {
```

Comment:
@astreet I'm not familiar with how it works, but I think it tries to load the bundle from cache if packager is not running, and gets emoty file, which leads to cannot find variable `__fbBatchedBridge` error. However I didn't try release mode, so yeah, it might be failing. I'll test ASAP.

Any suggestions on how to handle this correctly?